src/server/implementation/refreshTokens.ts

```diff
@@ -83,10 +136,25 @@ export async function validateRefreshToken(
     logWithLevel(LOG_LEVELS.ERROR, "Expired refresh token session");
     return null;
   }
-  if (refreshTokenDoc.firstUsedTime === undefined) {
-    await ctx.db.patch(refreshTokenDoc._id, {
-      firstUsedTime: Date.now(),
-    });
-  }
   return { session, refreshTokenDoc };
 }
+/**
+ * The active refresh token is the most recently created refresh token that has
+ * never been used.
+ *
+ * @param ctx
+ * @param sessionId
+ * @returns
+ */
+export async function loadActiveRefreshToken(
+  ctx: QueryCtx,
+  sessionId: GenericId<"authSessions">,
+) {
+  const refreshTokenDoc = await ctx.db
```

Comment:
What about returning directly vs. creating the variable and returning it?

Reply:
I definitely had some debug logging here at one point and cleaned it up without also cleaning up the pointless variable 🙈